### PR TITLE
session: use `batch` as the keyword for non-transactional DML

### PR DESCRIPTION
**Pull request by ekexium**
Signed-off-by: ekexium <ekexium@gmail.com>

### What problem does this PR solve?

Issue Number: ref #33485

Problem Summary:

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [x] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: session/nontransactional_test.go [hunk 1]
```diff
@@ -365,26 +365,26 @@ func TestNonTransactionalDeleteAlias(t *testing.T) {
...
 	tk := testkit.NewTestKit(t, store)
 
-	goodSplitStmts := []string{
-		"split on test.t1.a limit 5 delete t1.* from test.t as t1",
-		"split on a limit 5 delete t1.* from test.t as t1",
-		"split on _tidb_rowid limit 5 delete from test.t as t1",
-		"split on t1._tidb_rowid limit 5 delete from test.t as t1",
-		"split on test.t1._tidb_rowid limit 5 delete from test.t as t1",
-		"split limit 5 delete from test.t as t1", // auto assigns table name to be the alias
+	goodbatchStmts := []string{
```
Comment (sticnarf) [inline session/nontransactional_test.go:368]: Capitalize "batch"?

**Review by sticnarf — APPROVED** [review]
Don't forget to update the proposal https://github.com/pingcap/tidb/pull/33501

**Review by cfzjywxk — APPROVED** [review]

**Pull request merged**